doc/ping.xml

```diff
@@ -107,15 +107,27 @@ xml:id="man.ping">
     arbitrary number of “pad” bytes used to fill out the
     packet.</para>
     <para>
+    <command>ping</command> can also send IPv6 Node Information
+    Queries (RFC4620). Intermediate
+    <emphasis remap="I">hop</emphasis>s may not be allowed, because
+    IPv6 source routing was deprecated (RFC5095).</para>
+  </refsection>
+
+  <refsection xml:id="destination">
+    <info>
+      <title>DESTINATION</title>
+    </info>
+    <para>
     <command>ping</command> works with both IPv4 and IPv6. Using
     only one of them explicitly can be enforced by specifying
     <option>-4</option> or
     <option>-6</option>.</para>
     <para>
-    <command>ping</command> can also send IPv6 Node Information
-    Queries (RFC4620). Intermediate
-    <emphasis remap="I">hop</emphasis>s may not be allowed, because
-    IPv6 source routing was deprecated (RFC5095).</para>
+    NOTE: For when doing ping to a IPv6 link-local scope
```

Comment:
I'm not a native English speaker but it seems that the previous formulation ("For IPv6, when doing ping [...]") was more natural.